### PR TITLE
Upgraded nextjs and fixed virtualizer bug

### PR DESCRIPTION
**Pull request by Icyviolet23**
# Description

Upgraded Nextjs to version 13.0.6
Upgraded React to version 18.2.0
Upgraded React-dom to version 18.2.0
Upgraded react-virtual to version 3.0.0-beta.26

Closes #[REPLACE WITH ISSUE NUMBER]

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

I manually tested on the course tool and it seems to work lol. I fixed the list scrolling bug and the virtualizer works

**Test Configuration**:

- Node.js version:
- Python version:
- Desktop/Mobile:
- OS:
- Browser:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: frontend/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (Lhwhatever) [inline frontend/.vscode/settings.json:1]: personal project settings should not be in the commit

File: frontend/src/components/ScheduleSearch.tsx [hunk 2]
```diff
@@ -75,9 +75,9 @@ const CourseCombobox = ({
   });
 
   const filteredCourses = getCourses();
-  const rowVirtualizer = useVirtual({
-    size: filteredCourses.length,
-    parentRef: listRef,
+  const rowVirtualizer = useVirtualizer({
```
Comment (Lhwhatever) [inline frontend/src/components/ScheduleSearch.tsx:78]: seems to work

**Review by Lhwhatever — CHANGES_REQUESTED** [review]

**Review by Lhwhatever — APPROVED** [review]
LGTM

**Comment by Nicolapps** [comment]
Awesome, thanks!

**Pull request merged**